AI_ML/NLP/COUNT_VECTORIZER/COUNT_VECTORIZER.py

```diff
@@ -15,17 +19,16 @@ def COUNT_VECTORIZER(dc_inputs: list[DataContainer], params: dict) -> DataContai
         x -> the feature names
         y -> the word counts themselves
     """
-    if dc_inputs[0].type not in ["dataframe", "matrix"]:
-        raise ValueError(
-            f"unsupported DataContainer type passed to COUNT_VECTORIZER: {dc_inputs[0].type}"
-        )
+
+    if isinstance(default, DataFrame):
+        data = default.m.values
+    else:
+        data = default.m
 
     vectorizer = CountVectorizer()
+    X = vectorizer.fit_transform(data.flatten())
 
-    X = vectorizer.fit_transform(dc_inputs[0].m)
+    x = pd.DataFrame({"tokens": vectorizer.get_feature_names_out()})
+    y = X.toarray()
 
-    return DataContainer(
-        type="ordered_pair",
-        x=numpy.array(vectorizer.get_feature_names_out()),
-        y=X.toarray(),
-    )
+    return CountVectorizerOutput(tokens=DataFrame(df=x), word_count_vector=Matrix(m=y))
```

Comment:
This is fine for now, but we should change the matrix return to a vector (vector type exists in frontier, I guess I forgot to add it to the actual flojoy package)

Reply:
nice when you do add I'll update corresponding changes